### ISSUE TITLE
[BUG] EACCES error with `npm remove` on npm@7.7.0

### ISSUE THREAD

**Issue by erunion** [npm/cli#2928]
### Current Behavior:
```
$ npm remove react
npm ERR! code EACCES
npm ERR! syscall open
npm ERR! path /usr/local/package-lock.json
npm ERR! errno -13
npm ERR! Error: EACCES: permission denied, open '/usr/local/package-lock.json'
npm ERR!  [Error: EACCES: permission denied, open '/usr/local/package-lock.json'] {
npm ERR!   errno: -13,
npm ERR!   code: 'EACCES',
npm ERR!   syscall: 'open',
npm ERR!   path: '/usr/local/package-lock.json'
npm ERR! }
npm ERR! 
npm ERR! The operation was rejected by your operating system.
npm ERR! It is likely you do not have the permissions to access this file as the current user
npm ERR! 
npm ERR! If you believe this might be a permissions issue, please double-check the
npm ERR! permissions of the file and its containing directories, or try running
npm ERR! the command again as root/Administrator.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/jon/.npm/_logs/2021-03-23T21_03_16_479Z-debug.log
```

### Expected Behavior:
Package should be removed. 

Running `npm install -g npm@7.6` and re-running `npm remove react` I do not get this error.

### Steps To Reproduce:
1. npm install -g npm@7.7
2. npm remove `any package`
3. See error

### Environment:
- OS: OSX 10.14.6
- Node: 14.15.1
- npm: 7.7.0

**Comment by T00rk** [comment]
Same problem

**Comment by ruyadorno** [comment]
fixed in **[npm@7.7.1](https://github.com/npm/cli/releases/tag/v7.7.1)**, thanks for the quick report @erunion

**Comment by erunion** [comment]
ty @ruyadorno! glad it wasn't just me 😄
